trie/bintrie/binary_node.go

```diff
@@ -31,8 +31,11 @@ type (
 var zero [32]byte
 
 const (
-	NodeWidth = 256 // Number of child per leaf node
-	StemSize  = 31  // Number of bytes to travel before reaching a group of leaves
+	NodeWidth     = 256 // Number of child per leaf node
```

Comment:
that name needs to change, because in verkle, every node had a "width" of 256 but now it's only the stem node. So I suggest `StemNodeWidth`